python_ta/typecheck/type_store.py

```diff
@@ -1,10 +1,14 @@
 import astroid
 from astroid.builder import AstroidBuilder
 from collections import defaultdict
-from python_ta.typecheck.base import parse_annotations, class_callable, accept_failable
+from python_ta.typecheck.base import parse_annotations, \
+    class_callable, accept_failable, _node_to_type
 from typing import Callable
 import os
+from builtins import *
```

Comment:
Are these two imports still necessary? I think you removed the `globals()` call that had previously required it, if I understand correctly.